README.md

```diff
@@ -619,22 +617,22 @@ thread_b: Hello World from cpu 0 on xenvm!
 Detached from console
 ```
 
-Destroy domain - DomU will be destroyed and Xen domain id "3" will not be accessible any more.
-
+To destroy the DomU, run:
 ```
 uart:~$ xu destroy 3
 domain:3 destroyed
 ```
+After destroying the Xen domain, ID "3" will no longer be accessible.
```

Comment:
Again, this change doesn't comply with the previous steps. It could be better to drop it.